BEACON-V2-draft4-Model/genomicVariations/examples/genomicVariant-MID-example.json

```diff
@@ -4,10 +4,21 @@
   "variantType": "SNP",
   "referenceBases": "G",
   "alternateBases": "A",
-  "position": {
-    "assemblyId": "GRCh38",
-    "refseqId": "chr1",
-    "start": [55039979]
+  "location": {
+    "type": "SequenceLocation",
+        "interval": {
+          "type": "SequenceInterval",
+          "start": {
+            "type": "Number",
+            "value": 55039979
+          },
+          "end": {
+            "type": "Number",
+            "value": 55039980
+          },
+        },
+        "sequenceId": "refseq:NC_000001.11",
```

Comment:
```suggestion
        "sequence_id": "refseq:NC_000001.11",
```

Reply:
Hmmm... While I prefer snake_case for the data examples, too, there had been a change to `camelCase` throughout Beacon.